Datasnap/Classes/Event Properties/Place.h

```diff
@@ -9,7 +9,7 @@
 #import "EventProperty.h"
 
 @interface Place : EventProperty
-@property NSString* id;
+@property NSString* identifier;
```

Comment:
@almcintyre , @DevinJohnson, might be better to use a descriptive id name, like 'eventid'. I personally have not seen 'identifier' used before that I can recall